assets/src/blocks/SecondaryNavigation/SecondaryNavigationEditor.js

```diff
@@ -0,0 +1,72 @@
+import {getHeadingsFromBlocks} from '../TableOfContents/getHeadingsFromBlocks';
+import {initializeJustifyContentAdjustment} from './adjustNavWidth';
+
+const {useSelect} = wp.data;
+const {useEffect} = wp.element;
+const {InspectorControls} = wp.blockEditor;
+const {PanelBody} = wp.components;
+const {__} = wp.i18n;
+
+const renderEdit = () => {
```

Comment:
Very tiny suggestion, we can remove the `return` here and directly use the parentheses:
```
const renderEdit = () => (...);
```